subgraphs/uniswap-v2/schema.graphql

```diff
@@ -0,0 +1,464 @@
+enum Network {
```

Comment:
You should update this with the latest shared schema in the repo root

Reply:
Updated!